src/advanced/node-management.md

```diff
@@ -168,6 +168,33 @@ JSON object with the following fields:
 }
 ```
 
+### Healthcheck
+
+```none
+GET {system_base_path}/healthcheck
+```
+
+Returns a boolean representing if the node is connected to other peers.
+
+#### Parameters
+
+None.
+
+#### Response
+
+JSON object with the following fields:
+
+- **connectivity**: bool  
+  Is the node connected to other peers?
```

Comment:
I'd rather paraphrase it as a statement "Indicates whether the node is connected to the other peers."